content/posts/2022/retention-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Automatically delete workspace files with retention rules"
+date: 2022-11-29T10:30:00+01:00
+---
+
+It can be useful to be able to keep some files in your workflow's workspace only
+for a limited period of time, for example if the workflow generates huge
+temporary files that consume your
+[disk quota limit](https://docs.reana.io/advanced-usage/user-quotas). For this
+reason, it is now possible to define workspace file retention rules to configure
+the automatic deletion of files inside workspaces.
+
+<!--more-->
+
+You can define retention rules in the `reana.yaml` specification file of your
+workflow. For example, this is how you would specify that you would like to
+delete `tmp/*.root` files after seven days and `data/*.csv` files after thirty:
+
+```yaml
+workspace:
+  retention_days:
+    tmp/*.root: 7
+    data/*.csv: 30
+```
+
+Please note that files and directories specified as inputs or outputs of your
+workflow will not be deleted, even if they match one of the retention rules.
+This is to make sure that your workflows can be reproduced even after applying
+the retention rules.
+
+Once you run your workflow, you will be able to check the status of the rules
+through the web interface, where you will also find the scheduled deletion
+times:
+
+![Retention rules in the web UI](/images/ui-retention-rules.png)
+
+You can also use the new `retention-rules-list` command of `reana-client` to
+list the retention rules of a particular workflow:
+
+```console
+$ reana-client retention-rules-list -w reana-demo-root6-roofit
+WORKSPACE_FILES   RETENTION_DAYS   APPLY_ON              STATUS
+tmp/*.root        7                2022-12-06T23:59:59   active
+data/*.csv        30               2022-12-29T23:59:59   active
+```
+
+To learn more about retention rules and how they work, please take a look at the
+related
+[Workspace retention](https://docs.reana.io/advanced-usage/workspace-retention)
+documentation page.
+
+## Availability
+
+Workspace file retention rules will be part of the REANA 0.9.0 release series.
+However, you can already use them on the [reana.cern.ch](https://reana.cern.ch)
+instance. Please let us know what you think of this feature!
```

Comment:
The last phrase "Please let us know..." I would probably delete, since we did run an RFC with people on for this... It would be too late to ask for comments now. We can simply terminate with "...instance." Or say something like "Please let us know whether the new feature covers all your automated file deletion needs." But ending simply is perhaps preferable.